.insomnia/Request/req_b0483d6f0ea34b7eba74698e529ae1d1.yml

```diff
@@ -0,0 +1,147 @@
+_id: req_b0483d6f0ea34b7eba74698e529ae1d1
+type: Request
+parentId: fld_eeb62a531f8441478a6d2f536021400d
+modified: 1676745409437
+created: 1676743495518
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/copy"
+name: Copy
+description: >-
+  Copy articles from one review to another based on specified filters. 
+
+
+  Indiciate the target review to copy by specifying it in the request body:
+
+
+  - `target_review`: (review id to copy to)
+
+
+  Filters are passed in through the request body and are described below:
+
+
+  - `start`: Number of initial articles to skip when retrieving. Note that this depends on the sort order specified by the `order` parameter.
```

Comment:
Do we really support start, length and order in copying?

Reply:
Good point, we do not.